### PR TITLE
Added images for Python-Slim

### PR DESCRIPTION
**Pull request by orthros**
Added images for 1.13.0, 1.4 and 1.0 for Python based on python:*-slim

### CODE REVIEW

File: 1.0/python/slim/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2016, Google Inc.
```
Comment (mehrdada) [inline 1.0/python/slim/Dockerfile]: Who is "Google Inc."? When is 2016?

File: 1.4/python/slim/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 gRPC authors.
```
Comment (mehrdada) [inline 1.4/python/slim/Dockerfile]: "2018 The gRPC Authors" (no period)

File: 1.4/python/slim/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-slim
+
+ENV GRPC_PYTHON_VERSION 1.4.0
```
Comment (mehrdada) [inline 1.4/python/slim/Dockerfile:17]: Do we need to provide this for 1.4? I'd say keep this for new dockerfiles only

**Review by mehrdada — CHANGES_REQUESTED** [review]

File: 1.4/python/slim/Dockerfile (hunk 3 above)
Comment (orthros) [reply 1.4/python/slim/Dockerfile:17]: I added it for 1.4 to add parity/uniformity for all python images. We can remove the image if necessary

File: 1.4/python/slim/Dockerfile (hunk 2 above)
Comment (orthros) [reply 1.4/python/slim/Dockerfile]: Done

File: 1.0/python/slim/Dockerfile (hunk 1 above)
Comment (orthros) [reply 1.0/python/slim/Dockerfile]: I copied the license header information from the existing 1.0. I'll change it to "2018 The gRPC Authors"

**Review by mehrdada — APPROVED** [review]

**Comment by orthros** [comment]
Thanks!

**Pull request merged**